### PR TITLE
Img

### PR DESCRIPTION
**Pull request by codingoak**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,8 +18,31 @@ <h1>Daniel Eicher</h1>
...
+    <img src="https://source.unsplash.com/random/600x300" alt="Beispielbild">
+    <figcaption>Dies ist ein Beispielbild</figcaption>
+  </figure>
   <section id="contact">
     <h2>Contact</h2>
+    <ul>
+      <li><b>Phone: </b>+34 642 104 722</li>
+      <li><b>Mail: </b>danieleicher@gmx.net</li>
+    </ul>
+
```
Comment (JelenadlS) [inline index.html:31]: I think is looks good, nice idea :-)

File: index.html [hunk 2]
```diff
@@ -18,8 +18,31 @@ <h1>Daniel Eicher</h1>
...
+        src="https://upload.wikimedia.org/wikipedia/commons/9/91/Octicons-mark-github.svg" width="32px" height="32px"
+        alt="github-logo"></a>
+    <!-- CodePen-logo with a link to my profil-->
+    <a href="https://codepen.io/codingoak" target="_blank"><img
+        src="https://res.cloudinary.com/css-tricks/images/w_32,h_32,c_fill/f_auto,q_auto/v1637703542/Button-Black-Large/Button-Black-Large-png?_i=AA"
+        width="32px" height="32px" alt="codepen-logo"></a>
+    <!-- LinkedIn-logo with a link to my profil-->
+    <a href="https://www.linkedin.com/in/daniel-eicher-963844203/" target="_blank"><img
+        src="https://upload.wikimedia.org/wikipedia/commons/c/ca/LinkedIn_logo_initials.png" width="32px" height="32px"
+        alt="linkedin-logo"></a>
```
Comment (JelenadlS) [inline index.html:45]: I googled it and it seems to be correct. However, I didn't do it yet :-)

**Review by JelenadlS — COMMENTED** [review]
I think someone else should have another look to the Connect part :-)

File: index.html [hunk 3]
```diff
@@ -18,8 +18,31 @@ <h1>Daniel Eicher</h1>
       </ul>
     </nav>
   </header>
+  <figure>
+    <img src="https://source.unsplash.com/random/600x300" alt="Beispielbild">
+    <figcaption>Dies ist ein Beispielbild</figcaption>
```
Comment (ttaannjjaa) [inline index.html:23]: no code problem - but i believe, you want to create your personal website in English - then i would change this 🇬🇧

File: index.html [hunk 4]
```diff
@@ -18,8 +18,31 @@ <h1>Daniel Eicher</h1>
...
+  </figure>
   <section id="contact">
     <h2>Contact</h2>
+    <ul>
+      <li><b>Phone: </b>+34 642 104 722</li>
+      <li><b>Mail: </b>danieleicher@gmx.net</li>
```
Comment (ttaannjjaa) [inline index.html:29]: good idea to improve the contrast here , i am curious whether we are taught to do this the way you did or take CSS

File: index.html (hunk 2 above)
Comment (codingoak) [reply index.html:45]: I'm not sure about the logos. There are copyrights and rules to use it...

**Review by ttaannjjaa — CHANGES_REQUESTED** [review]
for me it looks very well 😃- cool things added and i like your additional comments for structuring and easier reading very much

File: index.html (hunk 3 above)
Comment (codingoak) [reply index.html:23]: thx a lot - I will fix it :)

File: index.html (hunk 2 above)
Comment (JelenadlS) [reply index.html:45]: Well yeah, that's a good one. Maybe we can ask the team as well as the coaches tomorrow  :-)

**Pull request merged**